tests/D2L.CodeStyle.Analyzers.Test/Async/Generator/AsyncToSyncMethodTransformerTests.cs

```diff
@@ -337,6 +337,15 @@ internal sealed class AsyncToSyncMethodTransformerTests {
 		Assert.AreEqual( @"[Blocking] Foo Bar() { var ct = new CancellationToken(); return Foo( 1,""test"" ); }", actual.Value.ToFullString() );
 	}
 
+	[Test]
+	public void CancellationTokenParameter() {
+		var actual = Transform( @"[GenerateSync] Task<Foo> BarAsync( int Baz, CancellationToken ct ) { return FooAsync( Baz + 1, ct ); }" );
+
+		Assert.IsTrue( actual.Success );
+		Assert.IsEmpty( actual.Diagnostics );
+		Assert.AreEqual( "[Blocking] Foo Bar( int Baz) { return Foo( Baz + 1); }", actual.Value.ToFullString() );
+	}

```

Comment:
This is good 👍 
We're going to have a separate PR to address invocation argument lists right?
Like a call to `await FooAsync( x, y, ct );` currently becomes `Foo( x, y, ct );` (I think) but we want it to be `Foo( x, y )`, right?
(Separate PR for that is good.)

Reply:
Oh wait nvm I see that's happening in this test, I'll read the PR more closely :)